docs/modules/ROOT/pages/how-tos/crossplane/working_with_functions.adoc

```diff
@@ -0,0 +1,197 @@
+= How to work with Crossplane Functions
+
+Crossplane introduced `Functions` in version v1.11 as an Alpha feature: https://docs.crossplane.io/knowledge-base/guides/composition-functions. 
+
+Function is a container to which is passed FunctionIO yaml object as stdin. FunctionIO object is created and managed by Crossplane to keep state of our resources. It has 3 main entries:
+
+* observed: this one describes what is current state of our resources, it's read-only, all changes will be discarded
```

Comment:
From what I have read, you can also optionally add another resource not related to the composition or XR. It has to be K8s compliant (kind and apiVersion) and goes straight to config field. I would add this info here as well.